FiducialRegistrationWizard/MRML/vtkMRMLFiducialRegistrationWizardNode.cxx

```diff
@@ -253,6 +289,107 @@ void vtkMRMLFiducialRegistrationWizardNode::SetUpdateMode( std::string newUpdate
 }
 
 //------------------------------------------------------------------------------
+std::string vtkMRMLFiducialRegistrationWizardNode::UpdateModeAsString( int updateMode )
+{
+  switch ( updateMode )
+  {
+    case UPDATE_MODE_MANUAL:
+    {
+      return "Manual";
+    }
+    case UPDATE_MODE_AUTO:
+    {
+      return "Auto";
+    }
+    default:
+    {
+      vtkGenericWarningMacro( "Unrecognized value for UpdateMode " << updateMode << ". Returning \"Manual\"" );
+      return "Manual";
+    }
+  }
+}
+
+//------------------------------------------------------------------------------
+int vtkMRMLFiducialRegistrationWizardNode::UpdateModeFromString( std::string updateModeAsString )
+{
+  for ( int updateMode = 0; updateMode < UPDATE_MODE_LAST; updateMode++ )
+  {
+    bool matchingText = updateModeAsString.compare( UpdateModeAsString( updateMode ) ) == 0;
+    if ( matchingText )
+    {
+      return updateMode;
+    }
+  }
+  // if there are no matches, then there is likely an error
+  vtkGenericWarningMacro( "Unrecognized string for UpdateMode " << updateModeAsString << ". Returning value for \"Manual\"" );
+  return UPDATE_MODE_MANUAL;
+}
+
+//------------------------------------------------------------------------------
+void vtkMRMLFiducialRegistrationWizardNode::SetPointMatchingMethod( int newMethod )
+{
+  if ( this->GetPointMatchingMethod() == newMethod )
+  {
+    // no change
+    return;
+  }
+  this->PointMatchingMethod = newMethod;
+  this->Modified();
+  this->InvokeCustomModifiedEvent(InputDataModifiedEvent);
+}
+
+//------------------------------------------------------------------------------
+std::string vtkMRMLFiducialRegistrationWizardNode::PointMatchingMethodAsString( int method )
+{
+  switch ( method )
+  {
+    case POINT_MATCHING_METHOD_INPUT_ORDER:
+    {
+      return "Input Order";
+    }
+    case POINT_MATCHING_METHOD_COMPUTED:
+    {
+      return "Computed";
```

Comment:
For me, it is not clear what you mean by "Input order". Also, there may be multiple computation methods in the future. So, the options could be:
- original order (not very good either, so ask a few people if they understand what we mean by this)
- automatic reorder

Reply:
Or even something as simple as "Manual order" vs "Automatic order". I'll try to gather a few ideas.

Reply:
I like manual / automatic ordering